src/functionHandlers/notarisePdt/handler.ts

```diff
@@ -78,9 +78,23 @@ export const main: Handler = async (
     };
   }
 
+  let result;
+
   try {
-    const result = await notarisePdt(reference, certificate);
+    result = await notarisePdt(reference, certificate);
+  } catch (e) {
+    const errorWithRef = error.extend(`reference:${reference}`);
```

Comment:
Small nit: now that there are > 1 instances of this logger, can initialize it once at the top to be used throughout the multiple catch blocks?

Please resolve this once fixed :)